application/src/main/java/org/togetherjava/tjbot/features/utils/LinkDetections.java

```diff
@@ -0,0 +1,83 @@
+package org.togetherjava.tjbot.features.utils;
+
+import com.linkedin.urls.Url;
+import com.linkedin.urls.detection.UrlDetector;
+import com.linkedin.urls.detection.UrlDetectorOptions;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Utility class to detect links.
+ */
+public class LinkDetections {
+
```

Comment:
shouldn't it named just `LinkDetection`?

Reply:
I wasnt sure because the other helper class is also called `LinkPreviews`

Reply:
imo LinkDetection would be a good fit here. Either way LinkDetections feels a bit odd to read.